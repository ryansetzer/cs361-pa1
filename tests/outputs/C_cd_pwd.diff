--- outputs/C_cd_pwd.txt	2022-09-25 14:55:14.654636675 -0400
+++ expected/C_cd_pwd-2.txt	2022-09-11 14:59:21.000000000 -0400
@@ -0,0 +1,7 @@
+$ cd /usr/bin
+$ pwd
+/usr/bin
+$ which ls
+/usr/bin/ls
+$ quit
+
