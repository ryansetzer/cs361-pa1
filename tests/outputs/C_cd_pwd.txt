$ cd /usr/bin
$ pwd
/usr/bin
$ which ls
/usr/bin/ls
$ quit

