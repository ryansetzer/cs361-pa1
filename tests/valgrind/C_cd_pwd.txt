==59330== Memcheck, a memory error detector
==59330== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==59330== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==59330== Command: ../dukesh -b scripts/dirs.txt
==59330== 
==59330== 
==59330== HEAP SUMMARY:
==59330==     in use at exit: 0 bytes in 0 blocks
==59330==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==59330== 
==59330== All heap blocks were freed -- no leaks are possible
==59330== 
==59330== For lists of detected and suppressed errors, rerun with: -s
==59330== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
