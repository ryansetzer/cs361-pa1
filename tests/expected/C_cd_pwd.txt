$ cd /usr/bin
$ pwd
/usr/bin
$ which ls
/bin/ls
$ quit

